tests/disaster-recovery/regional-dr/test_failover.py

```diff
@@ -57,16 +82,51 @@ def test_failover(
         logger.info(f"Waiting for {wait_time} minutes to run IOs")
         sleep(wait_time * 60)
 
+        primary_cluster_name = get_current_primary_cluster_name(
+            rdr_workload.workload_namespace
+        )
+        secondary_cluster_name = dr_helpers.get_current_secondary_cluster_name(
+            rdr_workload.workload_namespace
+        )
+
+        if config.RUN.get("rdr_failover_via_ui"):
+            ocs_version = version.get_semantic_ocs_version_from_config()
+            if ocs_version >= version.VERSION_4_13:
+                logger.info("Start the process of failover from ACM UI")
+                config.switch_acm_ctx()
+                dr_submariner_validation_from_ui(acm_obj)
+            else:
+                logger.error("ODF/ACM version isn't supported for Failover operation")
```

Comment:
As discussed, we can't have this change in conftest as the deployment still supports OCP 4.12 with ACM 2.6. So I moved the check to the top so that we could avoid running IOs and save some time if the version isn't supported.